doc-content/enterprise-only/employee-rostering/er-skills-proc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='er-skills-proc']
+= Entering skills
+
+You can set all _skills_ that can be necessary in any position within the roster. For example, a 24-hour diner would require cooking, serving, bussing, and hosting skills, in addition to skills such as general human resources and restaurant operations.
```

Comment:
Skill is in italic because it is a definition (just like spot is in italic in its own procedure). And "can be necessary" because the user decides if any particular skill is necessary. 

I changed "would" to "can", but I don't want to write "a diner requires" because this claims too much authority about diners.